public/js/monitor.js

```diff
@@ -441,40 +427,15 @@ function resizeDashboardMargin() {
       });
     }
   }
-  const researchPromo = document.querySelector(".research-recruitment-wrapper");
-  const sendRecruitmentPing = (action, interactionStatus, transportType) => {
-    if (gaAvailable) {
-      ga("send", "event", "Research Recruitment Promo", action, "Recruitment Promo", {nonInteraction: interactionStatus}, {transport: transportType});
-    }
-  };
-  const promoHasAlreadyBeenDismissed = (researchPromo && sessionStorage.getItem("fxMonitorResearchPromo") === "dismissed");
-  const sessionStorageDoesNotExist = (researchPromo && !sessionStorage);
-  const acceptedLanguages = navigator.languages;
-  const acceptedLocalesIncludesEnglish =
-    (acceptedLanguages.includes("en-US") || acceptedLanguages.includes("en"));
-
-  if (promoHasAlreadyBeenDismissed || sessionStorageDoesNotExist || !acceptedLocalesIncludesEnglish) {
-    researchPromo.style.display = "none";
-  }
 
   resizeDashboardMargin();
 
-  if (researchPromoIsVisible()) {
-    sendRecruitmentPing("View", true, "image");
-    document.querySelector(".x-close-promo").addEventListener("click", () => {
-      if (sessionStorage) {
-        sessionStorage.setItem("fxMonitorResearchPromo", "dismissed");
-        researchPromo.classList.add("hidden");
-      }
-      sendRecruitmentPing("Engage", false, "beacon");
-    });
-  }
-
   const dropDownMenu = document.querySelector(".mobile-nav.show-mobile");
   dropDownMenu.addEventListener("click", () => toggleDropDownMenu(dropDownMenu));
 
 
 
+  const acceptedLanguages = navigator.languages;
```

Comment:
@lesleyjanenorton Added this back as it's in use below.

Reply:
could be:
```
const fxaCheckbox = document.getElementById("fxaCheckbox");
if (!acceptedFirstLanguageIsEnglish && fxaCheckbox) {
  fxaCheckbox.style.display = "none" 
}